components/zcash_note_encryption/Cargo.toml

```diff
@@ -11,3 +11,13 @@ license = "MIT OR Apache-2.0"
 edition = "2018"
 
 [dependencies]
+blake2b_simd = "0.5"
+byteorder = "1"
+crypto_api_chachapoly = "0.4"
+ff = "0.8"
+group = "0.8"
+rand_core = "0.5.1"
+
+[dev-dependencies]
+zcash_primitives = { version = "0.5", path = "../../zcash_primitives" }
```

Comment:
These not-quite-circular dependencies make publishing a pain. I dealt with it for `zcash_client_backend <-> zcash_client_sqlite` for the 0.5 release by specifying a version range per https://github.com/rust-lang/cargo/issues/4242 but it would be good to formalize this in the repo. Non-blocking, we can sort this out later.

Reply:
With the zcash_client_backend issue, I'm planning to address that by making the tests just run against the in-memory backend once that's complete, and move the sqlite tests back to the sqlite crate. Here, I'm not sure what the best approach is; I guess we could move these tests to zcash_primitives and implement a set of tests here that just relies upon the note-encryption functionality without using Sapling.